js/private/js_binary.bzl

```diff
@@ -33,6 +33,9 @@ Bazel option to see more detail about the selection.
 All [common binary attributes](https://bazel.build/reference/be/common-definitions#common-attributes-binaries) are supported
 including `args` as the list of arguments passed Node.js.
 
+Node.js execution is performed by a shell script that sets environment variables and runs the Node.js binary with the `entry_point` script.
+The shell script is located relative to the directory containing the `js_binary` at `\\{name\\}_/\\{name\\}`.
```

Comment:
Perhaps link to the PR for more info on why that naming scheme was chosen: https://github.com/aspect-build/rules_js/pull/1690